mkdocs.yaml

```diff
@@ -5,7 +5,6 @@ edit_uri: edit/main/docs/
 nav:
   - Deployment:
       - Information: 'deployment/information.md'
-  - Read Me: '../../../README.md'
```

Comment:
The index file already has the content from the README, and the link shows up as a broken one in TechDocs due to a path issue.